### ISSUE TITLE
insider reflow validation: overflow button fails 'aria-required-children'

### ISSUE THREAD

**Issue by karanbirsingh** [microsoft/accessibility-insights-web#3075]
## Describe the bug

- open the details view with reflow (Insider 2.19)
- resize the window smaller so the top-right "..." button appears
- run A11YSelfValidator.validate()

failure:
description: "Ensures elements with an ARIA role that require child roles contain them"
id: "aria-required-children"
message: "Required ARIA children role not present: menuitem, menuitemradio, menuitemcheckbox"

We have a ['menu' role on the button](https://github.com/microsoft/accessibility-insights-web/blob/993566dd91aea3cf1cb1bfd021ad44eab8a87db3/src/DetailsView/components/command-bar-buttons-menu.tsx#L40), but the required ARIA children only show up when the button is pressed. So if the menu is collapsed, the child roles are not in the DOM.

Docs for CommandBar or ContextualMenu in case we want to inspect how they do it:
- https://developer.microsoft.com/en-us/fluentui/#/controls/web/commandbar
- https://developer.microsoft.com/en-us/fluentui/#/controls/web/contextualmenu

Perhaps a case for menubutton instead? https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton

## Context (please complete the following information)

- AI-Web Version & Environment: 2.19 Insider
- Browser Version: Version 83.0.4103.116 (Official Build) (64-bit)
